src/app/pages/backend-integration/backend-integration.component.scss

```diff
@@ -25,7 +25,7 @@
   max-width: 45rem;
   align-items: stretch;
   height: 100%;
-
+  
```

Comment:
Revert whitespace change. And configure your editor to trim trailing whitespace. Actually, ngx-admin has .editorconfig file, so your editor should pick up configured settings automatically. Do you have an editor config plugin? You use the VS Code, right? Here it is:
marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

Reply:
I use Webstorm, but i think it should pick configs too. Will check this out